src/test/functional/mutationServices.test/tagsMutationService-create.integration.ts

```diff
@@ -197,49 +207,49 @@ describe('Mutation for Tag: ', () => {
       query: createTagsMutation,
       variables,
     });
-    const expectedData = [
-      { name: 'bajeena', savedItems: [{ id: '0' }] },
-      { name: 'quattro', savedItems: [{ id: '0' }] },
-    ];
     expect(res.errors).to.be.undefined;
     const data = res.data.createTags;
     expect(data.length).to.equal(2);
-    expect(data).to.deep.equalInAnyOrder(expectedData);
+    expect(data[0].name).equals('bajeena');
+    expect(data[1].name).equals('quattro');
+    expect(data[0].savedItems.edges[0].node.id).equals('0');
+    expect(data[1].savedItems.edges[0].node.id).equals('0');
   });
   it('should handle emojis', async () => {
     const variables = {
       input: [{ savedItemId: 0, name: '🤪😒' }],
     };
-    const expectedData = [{ name: '🤪😒', savedItems: [{ id: '0' }] }];
     const res = await server.executeOperation({
       query: createTagsMutation,
       variables,
     });
     expect(res.errors).to.be.undefined;
-    expect(res.data.createTags).to.deep.equal(expectedData);
+    const data = res.data.createTags;
+    expect(data[0].name).equals('🤪😒');
+    expect(data[0].savedItems.edges[0].node.id).equals('0');
   });
 
   it('should handle other unicode, non-ascii chars', async () => {
     const variables = {
       input: [
         { savedItemId: 0, name: 'İnanç' },
         { savedItemId: 0, name: '𡞰' },
-        { savedItemId: 1, name: '(╯°□°)╯︵ ┻━┻' },
+        { savedItemId: 0, name: '(╯°□°)╯︵ ┻━┻' },
```

Comment:
This was intentionally for a different savedItem ID and should be reverted to cover this case

Reply:
reverted this test to have `1` for `(╯°□°)╯︵ ┻━┻` 

had it changed to 0 to easily loop through asserts on savedItems as we primarily test adding unicode tags in this test. but ya its good to retain the old test behavior.